package.json

```diff
@@ -84,6 +87,7 @@
     "rollup": "^2.0.0",
     "rollup-plugin-node-polyfills": "^0.2.0",
     "rollup-plugin-terser": "^7.0.0",
+    "typescript": "^4.0.3",
```

Comment:
```suggestion
    "typescript": "^4.0.0",
```

Reply:
is this even needed? doesn’t dtslint try with lots of different versions?

Reply:
It does try different versions,
but DTSLint made it a peerDependency in the latest version, and requires the peer to run, not entirely sure why :man_shrugging: 
https://github.com/microsoft/dtslint/blob/952086da5617f3f2c94bc9aad849fe51732eb00e/package.json#L40-L42

Reply:
oh, interesting. So we’ll hit this with other unified stuff too, good to know!